docs/articles/documentation/test-api/selecting-page-elements/examples-of-working-with-dom-elements.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: docs
+title: Examples of Working with DOM Elements
+permalink: /documentation/test-api/selecting-page-elements/examples-of-working-with-dom-elements.html
+---
+# Examples of Working with DOM Elements
+
+This document shows how to work with DOM elements in frequent real-world situations.
+
+* [Accessing Page Element Properties](#accessing-page-element-properties)
+* [Getting a Page Element Using Custom Logic](#getting-a-page-element-using-custom-logic)
+* [Checking if an Element is Available](#checking-if-an-element-is-available)
+* [Enumerating Elements Identified by a Selector](#enumerating-elements-identified-by-a-selector)
+
+## Accessing Page Element Properties
+
+To work with page elements, use TestCafe [selectors](selectors/README.md).
+Import the `Selector` function and call it passing a CSS selector inside.
+This function creates a selector object [whose API](dom-node-state.md) exposes the most used members of HTML element API.
```

Comment:
Change from
...object [whose API](dom-node-state.md)...
to 
...object whose [API](dom-node-state.md)...